.github/workflows/build_packages.yml

```diff
@@ -133,6 +133,7 @@ jobs:
       goarch: ${{ matrix.goarch }}
       package_arch: ${{ matrix.package_arch }}
       build_tool: ${{ matrix.build_tool }}
+      fips: ${{ matrix.fips || false }}
```

Comment:
Does `matrix.fips` not default to false when it's not specified?

Reply:
It defaults to empty string, and then the workflow fails because it expects a boolean.